vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -6,16 +6,11 @@
         This script is used by packer to produce Windows AKS images.
 #>
 
-param (
-    $containerRuntime,
-    $windowsSKU
-)
-
 $ErrorActionPreference = "Stop"
 
-filter Timestamp { "$(Get-Date -Format o): $_" }
+. c:\windows-vhd-configuration.ps1
```

Comment:
I find it's used in multiple places,  is it possible to create a variable for it?

Reply:
Only twice. One for import and another is for deletion. We can improve it if we think that it is worth later.